sdk/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -38,13 +39,24 @@
    * Span#updateName(String)} so this value cannot be cached.
    *
    * @return the name of the {@code Span}.
+   * @since 0.1.0
    */
   String getName();
 
   /**
-   * Returns the proto representation of the collected data for this particular {@code Span}.
+   * This method is here to convert this instance into a protobuf instance. It will be removed from
+   * this class soon, so if you are writing new code you should not use this method. It is left here
+   * to help reduce the number of simultaneous changes in-flight at once.
    *
-   * @return the proto representation of the collected data for this particular {@code Span}.
+   * @return a new protobuf Span instance.
    */
   io.opentelemetry.proto.trace.v1.Span toSpanProto();
```

Comment:
Nit: deprecate this method.

Reply:
I didn't deprecate it because that then requires a bunch of extra work in the classes that still use it. It'll go away entirely in the next step of this work. OK to defer and just delete it when we get there?

Reply:
I didn't deprecate it because that then requires a bunch of extra work in the classes that still use it. It'll go away entirely in the next step of this work. OK to defer and just delete it when we get there?

Reply:
Makes sense to me to defer to another PR.

Reply:
unsolicited 2p for open source:

deprecate would be the better call. you can't rely on people reading comments to know intent, and deprecation doesn't break the build later. plus what if something happens and you have to leave abruptly etc. deprecation shows the work is incomplete without relying on people reading this comment (out of hundreds of comments per week), so it is a good thing.